schemas/infoAssessment.json

```diff
@@ -116,7 +116,8 @@
                 "password": {
                     "description": "Password to begin the assessment (only for Exams).",
                     "type": "string"
-                }
+                },
+                "SEBconfig": { }
```

Comment:
`SEBconfig` should be `SEBConfig` (uppercase `C`). The sync code uses `SEBConfig`